cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BrokerStats.java

```diff
@@ -44,11 +44,12 @@ public BrokerStats() {
 
   public void addSingleBrokerStats(String host, int id, Broker.State state, double diskUtil, double cpuUtil, double leaderBytesInRate,
                                    double followerBytesInRate, double bytesOutRate, double potentialBytesOutRate,
-                                   int numReplicas, int numLeaders, boolean isEstimated, double capacity) {
+                                   int numReplicas, int numLeaders, boolean isEstimated, double capacity,
+                                   Map<String, Double> diskUtils, Map<String, Double> diskCapacities) {
```

Comment:
Are there any special cases worth mentioning about `diskUtils` and `diskCapacities` -- e.g. `null` when broker is `RAID`?

Reply:
It is will be null if `BrokerCapacityResolver` does not report per disk information, so it is not directly relevant with `RAID`.  Will update the Javadoc of `SingleBrokerStats`